src/MariaResult.cpp

```diff
@@ -1,19 +1,19 @@
-#include "pch.h"
 #include "MariaResult.h"
+
 #include "MariaResultPrep.h"
 #include "MariaResultSimple.h"
-
+#include "pch.h"
```

Comment:
Why did the "include" move? The `pch.h` include must come first here.

Reply:
Clang-format reordering header includes by some rules. But I think it's more general question. Seems `pch.h` is not `real` precompiled header, it's more like general include. 

But it cause to problem with order of includes, so if order is changes then build is failing.

Maybe it is make sense to fix includes, that they do not depend on the order?

Reply:
We could specify `IncludeCategories` so that `pch.h` is always sorted first?